src/widgets/configure/configure.ts

```diff
@@ -0,0 +1,21 @@
+import { PlainSearchParameters } from 'algoliasearch-helper';
+import connectConfigure from '../../connectors/configure/connectConfigure';
+import { WidgetFactory } from '../../types';
+
+/**
+ * A list of [search parameters](https://www.algolia.com/doc/api-reference/search-api-parameters/)
+ * to enable when the widget mounts.
+ */
+type ConfigureWidgetParams = PlainSearchParameters;
+
+type Configure = WidgetFactory<ConfigureWidgetParams>;
+
+const configure: Configure = (searchParameters: ConfigureWidgetParams) => {
```

Comment:
```suggestion
const configure: Configure = (widgetParams: ConfigureWidgetParams) => {
```

IMO it would be simpler to understand.

Reply:
Since we already use this pattern in other connectors tests, I think this can wait until we have a proper widget definition interface with only required lifecycle hooks. WDYT?